app/components/forms/wizard/basic-details-step.js

```diff
@@ -436,7 +436,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
 
     openTaxModal(isNewTax) {
       if (!this.isCreate && isNewTax) {
-        this.set('data.event.tax', this.store.createRecord('tax'));
+        this.set('data.event.tax', this.store.createRecord('tax', {name: this.data.event.tax.name}));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4922891)